src/pages/settings/Payments/PaymentsPage/BasePaymentsPage.js

```diff
@@ -441,7 +441,9 @@ class BasePaymentsPage extends React.Component {
                     shouldShowCancelButton
                     danger
                 />
-                <OfflineWithFeedback onClose={() => PaymentMethods.clearWalletError()} errors={this.props.userWallet.errors} errorRowStyles={[styles.ph6, styles.pv2]} />
+                <OfflineWithFeedback onClose={() => PaymentMethods.clearWalletError()} errors={this.props.userWallet.errors} errorRowStyles={[styles.ph6, styles.pv2]}>
+                    <div />
+                </OfflineWithFeedback>
```

Comment:
This seems wrong, `<div>` is not supported in native platforms. We should wrap whatever component this error is related to. If this is a generic error, we should use [DotIndicatorMessage](https://github.com/Expensify/App/blob/641b06cd46d420daf0861faade6dbf1a119e4487/src/components/DotIndicatorMessage.js) instead